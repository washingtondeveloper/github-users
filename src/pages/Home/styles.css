.gs-home {
  padding-top: 15px;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
}
