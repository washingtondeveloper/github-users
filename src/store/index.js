import { createStore, applyMiddleware } from "redux";
import reduxThunk from "redux-thunk";

import rootReducer from "./ducks";

export default createStore(rootReducer, applyMiddleware(reduxThunk));
