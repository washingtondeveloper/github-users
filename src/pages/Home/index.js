import React from "react";

import "./styles.css";

import User from "@/components/User";
import Repos from "@/components/Repos";
import Search from "@/components/Search";

export default function Home() {
  return (
    <div className="gs-home">
      <Search />
      <User />
      <Repos />
    </div>
  );
}
