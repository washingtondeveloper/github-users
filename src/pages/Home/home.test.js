import React from "react";

import renderer from "react-test-renderer";
import { Provider } from "react-redux";
import store from "../../store";

import Home from "./";

describe("Component Home", () => {
  test("Deve renderizar sem erros", () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Home />
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
