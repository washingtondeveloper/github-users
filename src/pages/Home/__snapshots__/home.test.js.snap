// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Home Deve renderizar sem erros 1`] = `
<div
  className="gs-home"
>
  <div
    className="input-group mb-2"
  >
    <input
      className="form-control"
      id="inlineFormInputGroup"
      onChange={[Function]}
      placeholder="GitUser"
      type="text"
      value=""
    />
    <div
      className="input-group-prepend"
    >
      <button
        className="btn btn-primary"
        disabled={true}
        onClick={[Function]}
      >
        <i
          aria-hidden="true"
          className="fa fa-search"
        />
      </button>
    </div>
  </div>
  <div
    className="card"
  >
    <img
      alt="background"
      className="card-img-top"
      src="https://mdbootstrap.com/img/Photos/Others/images/43.jpg"
    />
    <img
      alt="user"
      className="user-image"
      src="git.png"
    />
    <div
      className="card-body"
    >
      <h5
        className="card-title text-center"
      >
        <div
          className="css-mkgss9-BallBeat"
        >
          <div />
          <div />
          <div />
        </div>
      </h5>
    </div>
  </div>
  <ul
    className="list-group"
  />
</div>
`;
